azext_iot/tests/digitaltwins/__init__.py

```diff
@@ -528,6 +528,15 @@ def delete_servicebus_resources(self):
     def delete_user_identity(self):
```

Comment:
nit (I think) - I know this method is only called during teardown - but it seems strange for this to remove `all` user-assigned identities from all `dt` instances - do we need to specify a specific identity in the embedded call, or could we rename the function `delete_all_user_identities` or something?

Reply:
right now, I am only making one user identity (and thus deleting one user identity) and the tests that use user identities also only create one dt instance

the only reason for the for loop to remove it from all dt instances is for future proofing if there is a test that makes multiple dt instances.

Since there is only one user identity, saying `delete_all_user_identities` is awkward. I don't think there will be a test scenario with multiple user identities in the near future either.